database/seeds/ModelSeeders/UserSeeder.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Faker\Factory as Faker;
+
+class UserSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // DB::table('phpbb_users')->delete();
+        // DB::table('osu_user_stats')->delete();
+        // DB::table('osu_user_stats_fruits')->delete();
+        // DB::table('osu_user_stats_mania')->delete();
+        // DB::table('osu_user_stats_taiko')->delete();
+        // DB::table('osu_user_performance_rank')->delete();
+
+        $this->beatmapCount = App\Models\Beatmap::count();
+        $this->faker = Faker::create();
+        if ($this->beatmapCount === 0) {
+            $this->command->info('Can\'t seed events, scores, events or favourite maps due to having no beatmap data.');
+        }
+
+        // Store some modifiers
+        $this->improvement_speeds = [
+            (rand(100, 110) / 100), // Fast Learner
+            (rand(100, 102) / 100), // Slow Learner
+            (rand(100, 115) / 100), // Genius / Multiaccounter :P
+        ];
+
+        // Create 10 users and their stats
+        factory(App\Models\User::class, 10)->create()->each(function ($u) {
+
+            // USER STATS
+            $common_countries = ['US', 'JP', 'CN', 'DE', 'TW', 'RU', 'KR', 'PL', 'CA', 'FR', 'BR', 'GB', 'AU'];
+            $country_code = $common_countries[array_rand($common_countries)];
+
+            $rank1 = rand(1, 500000);
```

Comment:
if you don't want to loop it at least change this to actually correlate with `$st` numbers. It's currently off by one (`rank1` is for `st`, `rank2` is for `st1`, etc).